settings/src/lib.rs

```diff
@@ -151,8 +196,7 @@ pub struct ExitTunnelSettings {
 pub struct RitaSettingsStruct {
     payment: PaymentSettings,
     network: NetworkSettings,
-    #[serde(skip_serializing_if = "Option::is_none")]
-    exit_client: Option<ExitClientSettings>,
+    exit_client: ExitClientSettings,
```

Comment:
What's the point of a separate `ExitClientSettings` struct if it's not an Option here?

Reply:
I like to segregate stuff which make sense as a single struct, and so that methods can be implemented on it

Reply:
How is putting 4 fields in a separate struct justified? How could it possibly serve readability?

Reply:
Hrm, the intention was so that when you say jumped from somewhere that used this struct, you would only find relevant fields and methods to what you were looking for, and also implementing methods on the top level struct is a little painful, as you have to do .read().unwrap() on every access to self. With jump to definition and things like that, I don't think the cons are that bad

Reply:
@jkilpatr any thoughts on this issue?

Reply:
Okay, my argument is invalid, there's other member structs which would also need to follow the same rules for consistency if we were to go down that path (`payment`, `network` members). And that for real could make stuff hard to read.